front/pages/api/w/[wId]/labs/transcripts/index.ts

```diff
@@ -60,13 +62,10 @@ async function handler(
     // TODO: This should be a proper list operation.
     case "GET":
       const transcriptsConfigurationGet =
-        await LabsTranscriptsConfigurationResource.findByUserWorkspaceAndProvider(
-          {
-            auth,
-            userId,
-            provider: req.query.provider as LabsTranscriptsProviderType,
-          }
-        );
+        await LabsTranscriptsConfigurationResource.findByUserWorkspace({
```

Comment:
Based on this we expect only one configuration right?

Reply:
Yes